tests/dialects/test_dialect.py

```diff
@@ -102,14 +102,13 @@ def test_get_or_raise(self):
         lowercase_mysql = Dialect.get_or_raise("mysql, normalization_strategy = lowercase")
         self.assertEqual(lowercase_mysql.normalization_strategy.value, "LOWERCASE")
 
-        with self.assertRaises(ValueError) as cm:
-            Dialect.get_or_raise("mysql, normalization_strategy")
-
-        self.assertEqual(
-            str(cm.exception),
-            "Invalid dialect format: 'mysql, normalization_strategy'. "
-            "Please use the correct format: 'dialect [, k1 = v2 [, ...]]'.",
-        )
+        for dialect in ("mysql", "presto", "bigquery", "databricks"):
+            default_dialect = Dialect.get_or_raise(dialect)
+            lowercase_dialect = Dialect.get_or_raise(f"{dialect}, normalization_strategy")
+            self.assertEqual(
+                lowercase_dialect.normalization_strategy.value,
+                default_dialect.normalization_strategy.value,
+            )
```

Comment:
Let's just keep the old test and change `ValueError` -> `AttributeError`. I don't think allowing `normalization_strategy` without value makes sense, it's most certainly a user error